assets/snapshot.go

```diff
@@ -14,30 +14,30 @@ package assets
 
 import (
 	"context"
-	"errors"
 	"sort"
 
 	"code.vegaprotocol.io/vega/libs/proto"
 	"code.vegaprotocol.io/vega/types"
 )
 
 var (
-	activeKey  = (&types.PayloadActiveAssets{}).Key()
-	pendingKey = (&types.PayloadPendingAssets{}).Key()
+	activeKey         = (&types.PayloadActiveAssets{}).Key()
+	pendingKey        = (&types.PayloadPendingAssets{}).Key()
+	pendingUpdatesKey = (&types.PayloadPendingAssetUpdates{}).Key()
 
 	hashKeys = []string{
 		activeKey,
 		pendingKey,
```

Comment:
You need to add `pendingUpdatesKey` to this slice.

Reply:
addressed